### DISCUSSION TITLE
Json Encoder

### DISCUSSION THREAD

**Discussion by Sivasundareswaran** [simdjson/simdjson#1918] [Q&A]
Hi Team,

I am able to do the decode the JSON string using simdijson but not able to found the reference for  encode the json string using simdjson ,
please share some c++ reference for encoding json .

Thanks in Advance

**Comment by lemire** [comment]
Currently, simdjson provides a parser. It does not provide a general-purpose serialization framework.

There is an open issue on this topic:
https://github.com/simdjson/simdjson/issues/1386
